Gems/RosRobotSample/README.md

```diff
@@ -1,9 +1,38 @@
-# ROS 2 Robot Sample
+[![Apache License, Version 2.0][apache_shield]][apache]
 
-This gem contains a sample robot asset, ready to be used with the ROS 2 Gem. 
+# RosRobotSample Gem for Open 3D Engine (O3DE)
 
-The robot is delivered as a ready to use prefab containing models, physics and following ROS 2 components:
-- Frame
-- Robot Control
-- Twist Control
-- Lidar Sensor
+## Introduction
+
+This Gem contains a set of models and prefabs suitable for robotic simulations. More details about such simulations in [Open 3D Engine](https:://o3de.org) can be found in [ROS2 Gem documentation](https://github.com/o3de/o3de-extras/tree/development/Gems/ROS2). 
+
+Please refer to [O3DE documentation](https://docs.o3de.org/docs/user-guide/gems/) to learn more about Gems and about registering Gems in the system and O3DE projects.
+
+## Requirements
+- Any O3DE project with the [O3DE ROS2 Gem](https://github.com/o3de/o3de-extras/tree/development/Gems/ROS2) enabled.
+
+## Description
+This is an Asset Gem. It contains the [Husarion ROSbot XL](https://husarion.com/manuals/rosbot-xl/) robot - an autonomous mobile robot platform equipped with a 4x4 drive, allowing for trouble-free expansion with optional devices such as LIDAR, RGB-D cameras or a manipulator developed by [Husarion](https://husarion.com). The robot is delivered as a ready-to-use prefab, `ROSbot.prefab`, containing visual models, physics and the following ROS 2 components:
```

Comment:
Should not involve marketing jargon such as "trouble-free expansion", especially since the robot model has a set configuration and does not include a manipulator.

Focus more on users please - what they can do with it and how.